### ISSUE TITLE
different results when comparing Annex60 - IDEAS

### ISSUE THREAD

**Issue by Mathadon** [lbl-srg/BuildingsPy#117]
When re-running unit tests in IDEAS I'm getting different result than the same model in the Annex 60 library.

E.g.: 
![screen shot 2017-01-25 at 09 34 14](https://cloud.githubusercontent.com/assets/5970879/22283304/c6dd802e-e2e1-11e6-9a33-67e4a7c2b601.png)

```
*** Warning: CalendarTimeMonths.mat: calTim.day has absolute and relative error = 1.000e+00, 2.500e-01.
             Maximum error is at t = 259200.0

*** Warning: CalendarTimeMonths.mat: calTim.weekDay has absolute and relative error = 1.000e+00, 1.429e-01.
             Maximum error is at t = 259200.0

*** Warning: CalendarTimeMonths.mat: calTim.hour has absolute and relative error = 2.300e+01, 8.333e-02.
             Maximum error is at t = 259200.0

*** Warning: CalendarTimeMonths.mat: calTim.minute has absolute and relative error = 6.000e+01, 0.000e+00.
             Maximum error is at t = 216000.0

             For IDEAS_Utilities_Time_Validation_CalendarTimeMonths.txt,
             accept new file and update reference files? (Close plot window to continue.)
```

When I run the same models in Dymola I get:

![screen shot 2017-01-25 at 09 35 04](https://cloud.githubusercontent.com/assets/5970879/22283329/e488ca52-e2e1-11e6-8471-b1dcddace5d5.png)

These plots suggest that Dymola and buildingspy interpret the results differently. Buildingspy does not seem to be consistent in whether it takes the result value from before/after the event.

Is this easy to fix?

edit: To be clear, I get no errors when I run the Annex60 unit test with the same reference results as IDEAS.

**Comment by mwetter** [comment]
@tsnouidui Can you please check that events are properly dealt with in your current revision of BuildingsPy.

@Mathadon Could it be that you use a different version of BuildingsPy when you run Annex60 or IDEAS? I do not see a reason why the same python code and the same Modelica code (other than replacing Annex60 with IDEAS) should get different results.

**Comment by tsnouidui** [comment]
Yes I will.

**Comment by Mathadon** [comment]
I don't think so. The unit test script itself is also identical.

I had a deeper look at what happened. I think I made a mistake yesterday when copying reference results from one file to another since I cannot reproduce the results that lead to my conclusion of above.

It looks now like I stored wrong results a few weeks back using an older version of buildingspy or something. This can be closed. Sorry for the overhead! I'll make it up soon =)
